src/background.js

```diff
@@ -247,6 +256,10 @@ class Background {
     }
     return message;
   }
+
+  fetchCurrentUser() {
+    return this.currentUser ? Promise.resolve(this.currentUser) : this.gitlab.getCurrentUser();
```

Comment:
**help wanted**: not sure is this type of dummy cache is acceptable, probably I should add smthng like `BackgroundCache` and keep the current user instance in the storage?

Reply:
make sense, thx. 

... also this approach reduced the number of changes